pkg/docker/docker.go

```diff
@@ -544,7 +546,8 @@ func runContainerTar(opts RunContainerOptions, imageMetadata *docker.Image) (cmd
 	return cmd, tarDestination
 }
 
-// this funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
+// runContainerDockerRun funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
+// This function runs forever, ensure a sig handler is installed elsewhere to handle cntrl-c.
 func runContainerDockerRun(container *docker.Container, d *stiDocker, image string) {
```

Comment:
This has historically been a hard to understand function, at least for me :smile: 

It looks like it does nothing apart from calling `glog.Info`. One has to read and spend some time on the caller to figure out how this can possibly work.

The function name and documentation comment is sort of misleading, or at least we need to talk about it :ok_hand: